apps/pie-storybook/CHANGELOG.md

```diff
@@ -58,7 +58,7 @@
   - @justeattakeaway/pie-divider@0.13.4
   - @justeattakeaway/pie-form-label@0.13.4
   - @justeattakeaway/pie-icon-button@0.28.5
-  - @justeattakeaway/pie-input@0.18.2
+  - @justeattakeaway/pie-text-input@0.18.2
```

Comment:
I think this is probably inaccurate because these old versions of `pie-text-input` won't exist on npm